src/main/java/com/epam/eighty/repository/TopicRepository.java

```diff
@@ -1,16 +1,19 @@
 package com.epam.eighty.repository;
 
-import com.epam.eighty.domain.Topic;
+import java.util.List;
 
-import org.springframework.data.domain.Slice;
 import org.springframework.data.neo4j.annotation.Query;
 import org.springframework.stereotype.Repository;
 
+import com.epam.eighty.domain.Topic;
+
 /**
  * @author Aliaksandr_Padalka
  */
 @Repository("topicRepo")
 public interface TopicRepository extends BaseRepository<Topic, Long> {
+    
     @Query(value = "MATCH (root:`Topic`)-[:`contains`*]->(topic:`Topic`) WHERE ID(topic) = {0} RETURN root", elementClass = Topic.class)
-    Slice<Topic> getRootTopicsForTopic(Long id);
```

Comment:
I would name the method just "getRootTopic". And also it's better to change the query according to the new method name.